src/plugins/discover/public/application/components/table/table_row.tsx

```diff
@@ -78,7 +79,13 @@ export function DocViewTableRow({
   });
 
   return (
-    <tr key={field} data-test-subj={`tableDocViewRow-${field}`}>
+    <tr
+      key={field}
+      data-test-subj={`tableDocViewRow-${field}`}
+      aria-label={i18n.translate('discover.tble.tableDocViewRowLabel', {
```

Comment:
While this `tr` doesn't need an `aria-label`, all of the buttons inside it will

Reply:
Deleted the `aria-label` for `<tr>`, and verified all of the buttons inside it already have `aria-label` defined.